apps/cowswap-frontend/src/modules/application/containers/App/index.tsx

```diff
@@ -83,6 +88,12 @@ export function App() {
   const { hideNetworkSelector } = useInjectedWidgetParams()
   const { pendingActivity } = useCategorizeRecentActivity()
   const isMobile = useMediaQuery(Media.upToMedium(false))
+  const customTheme = useMemo(() => {
+    if (ACTIVE_CUSTOM_THEME === CustomTheme.HALLOWEEN && darkMode && featureFlags.isHalloweenEnabled) {
+      return 'darkHalloween' as CowSwapTheme
+    }
+    return undefined
+  }, [darkMode])
```

Comment:
Feature flag was created.

I did observe a bug:
1. Turn off the flag
2. Open the page on dark mode
3. Turn on the flag
4. Refresh the page
* Halloween is not activated

I think it's because of this:

Need to add the flag to the deps array.

```suggestion
  }, [darkMode, featureFlags.isHalloweenEnabled])
```

Reply:
Adjust eslint rule to avoid the problem:
https://github.com/cowprotocol/cowswap/pull/5048

Reply:
Addressed